test-interface/src/main/scala/org/scalajs/testinterface/internal/FrameworkLoader.scala

```diff
@@ -7,8 +7,10 @@ import sbt.testing.Framework
 private[internal] object FrameworkLoader {
 
   def loadFramework(frameworkName: String): Framework = {
+    val exportsNamespace =
+      scala.scalajs.runtime.environmentInfo.exportsNamespace
```

Comment:
Even if it wasn't, it wouldn't be an issue. Linking the test-interface compiled by 0.6.13 will require a 0.6.13 linker anyway, which will provide you with an `exportsNamespace`.